src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
+	static String copyright = "Team Blue";
+  static String version = "v1.0";
+  static boolean exit_prompt = false;
+
+	static Scanner scanner = new Scanner(System.in).useDelimiter(";");
+
+  public String readStatement(String input) throws IOException {
```

Comment:
Remove the `input` parameter. It's not used, and `readStatement` should **not** have any parameters anyway.